### ISSUE TITLE
Investigate how the cache is cleared for Liveblogs & Liveblog API when a block is deleted or edited in CAPI

### ISSUE THREAD

**Issue by OllysCoding** [guardian/frontend#24811]
We need to understand how the cache is cleared when a live block is updated or deleted.

An event occurred on the 25/03/2022 where a block had to be urgently deleted for legal reasons from the following liveblog: https://www.theguardian.com/politics/live/2022/mar/25/uk-politics-live-grant-shapps-p-and-o-ferries-boss-should-resign-cost-of-living-crisis-boris-johnson

The block persisted on the page even after CP had manually purged the cache for the liveblog page.

It appeared to be because a cache also sits in front of the liveblog API ~ This meant that when you loaded the page and the liveblog queried the API, it was getting given the cached deleted block, and immediately inserting it in the page.

We should investigate whether we do, and if so how, purge the cache when live blocks are updated or deleted. We should ensure that whatever method is used clears the cache for both the liveblog page and the liveblog API

**Comment by OllysCoding** [comment]
Should be paired on with Ed Tools!

**Comment by OllysCoding** [comment]
Closing this card for the time being.

We spent some time investigating in edtools but weren't able to determine the cause if this particular instance, but I can confirm that when a liveblock is deleted we would expect both the page and API cache to be cleared. 

As a lot of time has passed, we also now will have lost any logs in logs.gutools which are only persisted for 2 weeks, making it exceptionally hard to further investigate.

It may have been that this incident was a one off, and we believe that clearing the API cache manually (instructions for which were passed onto CP) solved the problem.
